src/cascadia/QueryExtension/ExtensionPalette.idl

```diff
@@ -36,7 +36,16 @@ namespace Microsoft.Terminal.Query.Extension
         Windows.UI.Xaml.Controls.IconElement ResolvedIcon { get; };
 
         event Windows.Foundation.TypedEventHandler<ExtensionPalette, IInspectable> ActiveControlInfoRequested;
-        event Windows.Foundation.TypedEventHandler<ExtensionPalette, IInspectable> AIKeyAndEndpointRequested;
         event Windows.Foundation.TypedEventHandler<ExtensionPalette, String> InputSuggestionRequested;
     }
+
+    [default_interface] runtimeclass TerminalContext : IContext
+    {
+        TerminalContext(String activeCommandline);
+    }
+
+    [default_interface] runtimeclass SystemResponse : IResponse
```

Comment:
Why will the extension palette create its own kind of Response? It's supposed to _consume_ responses, not produce them.

Reply:
In that case, we don't need to put it on the idl. Nobody outside of ExtensionPalette will create one. We can just use `winrt::implements` on an internal class.